php/variant-apache.md

```diff
@@ -0,0 +1,40 @@
+## `%%IMAGE%%:<version>-apache
```

Comment:
missing closing backtick :open_mouth: :wink:

Reply:
I blame the fellow I stole most of this content from while it was still a WIP on his computer. :wink: :smile: